docker-compose.dev.yaml

```diff
@@ -274,113 +274,21 @@ services:
       postgres-db:
         condition: service_started
         required: true
+      vllm:
+        condition: service_started
+        required: false
+      vllm-cpu:
+        condition: service_started
+        required: false
       otel-collector:
         condition: service_started
         required: true
 
-  vllm1:
```

Comment:
I think we should still retain these instances for development testing, especially when benchmarking the integrations of inference potentially other serving frameworks